bundles/org.eclipse.e4.ui.dialogs/src/org/eclipse/e4/ui/dialogs/filteredtree/FilteredTree.java

```diff
@@ -102,21 +102,40 @@ public class FilteredTree extends Composite {
 	 */
 	private static final long SOFT_MAX_EXPAND_TIME = 200;
 
+	/**
+	 * Time for refresh job delay in terms of expansion in long value
+	 */
+	private final long refreshJobDelay;
```

Comment:
I do not understand what "in terms of expansion in long value" means.

In addition, it would be good to have the variable reflect the unit, so either call it something like `refreshJobDelayInMillis` or, even better, use a `Duration` instance.